pages/common/coproc.md

```diff
@@ -0,0 +1,28 @@
+# coproc
+
+> BASH builtin for creating interactive asynchronous subshells.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#Coprocesses>.
+
+- Run a subshell asynchronously:
+
+`coproc { {{commands}}; }`
+
+- Create a named coprocess:
+
+`coproc {{name}} { {{commands}}; }`
+
+- Write to a named coprocess stdin:
+
+`echo "{{input}}" >&"${{{name}}[1]}"`
```

Comment:
```suggestion
- Write to a specific coprocess `stdin`:

`echo "{{input}}" >&"${{{name}}[1]}"`
```
Almost the same issue as above.
Optional fix: `stdin` can be emphasized with backticks.